### PR TITLE
New Navigation

### PR DESCRIPTION
**Pull request by Eshchock1**
1. Added user name and team name beside the avatar
2. Added event location onto event cards
3. Replaced old drawer navigation with bottom navigation (featuring home/summary, scouting, and settings).
4. Created new filler pages for summary, settings, as well as match details.
5. Created 2 stack navigators, the first for events to matches, and the seconds from matches to match details.
6. Passed match info as props to match details pages to be able to display the proper match info.
7. Changed stack navigation animation to a clean slide from right.

### CODE REVIEW

File: src/navigation/AppNavigator.js [hunk 1]
```diff
@@ -1,16 +1,65 @@
...
+
+import Summary from '../screens/Summary';
+import * as theme from '../constants/theme';
+
+import FontAwesome5Icon from 'react-native-vector-icons/FontAwesome5';
+import IoniconsIcon from 'react-native-vector-icons/Ionicons';
+import MaterialCommunityIconsIcon from 'react-native-vector-icons/MaterialCommunityIcons';
+
+
+// old drawer navigation
```
Comment (jeffrey-dot-li) [inline src/navigation/AppNavigator.js]: Get rid of this

File: src/navigation/AppNavigator.js [hunk 2]
```diff
@@ -1,16 +1,65 @@
...
+// const routeConfigs = createDrawerNavigator({
+//   Events: {screen: Events},
+//   Matches: {screen: Matches},
+//   },
+//   {
+//   initialRouteName: 'Events',
+//   });
```
Comment (jeffrey-dot-li) [inline src/navigation/AppNavigator.js]: Get rid of this

File: src/screens/Events.tsx [hunk 3]
```diff
@@ -13,12 +13,14 @@ import {types, actions} from '../store';
...
 import {connect} from 'react-redux'; 
 
+
 const getEvents = actions.events.getEvents;
 
 interface OwnProps {
   type: string;
   user: any;
   chart: Array<number>;
+  navigation:any;
```
Comment (jeffrey-dot-li) [inline src/screens/Events.tsx]: Specify navigation type

**Review by jeffrey-dot-li — APPROVED** [review]

File: src/App.tsx [hunk 4]
```diff
@@ -18,9 +18,9 @@ const App: () => React$Node = () => {
   return (
     <>
       <Provider store={store}>
-        <View style={styles.container}>
+        <SafeAreaView style={styles.container}>
```
Comment (seendsouza) [inline src/App.tsx]: Don't do this. Instead do wrap View with a SafeAreaView component with the `flex: 1` property like this
```jsx
<SafeAreaView style={{flex: 1}} >
    <View style={styles.container}>
        ...
    </View>
</SafeAreaView>
```

File: src/components/Card.tsx [hunk 5]
```diff
@@ -34,9 +35,13 @@ const Card = (props: CardProps) => {
         </Block>
       </Block>
       <Block flex={0.75} column middle>
-        <Text h3 style={{paddingVertical: 8}}>
+        <Text h3 style={{}}>
```
Comment (seendsouza) [inline src/components/Card.tsx]: Remove `style={{}}`.

File: src/components/Card.tsx [hunk 6]
```diff
@@ -34,9 +35,13 @@ const Card = (props: CardProps) => {
...
           {props.rightHeader}
         </Text>
+        
+        <Text light style={{}}>
```
Comment (seendsouza) [inline src/components/Card.tsx]: Remove `style={{}}`

File: src/navigation/AppNavigator.js [hunk 7]
```diff
@@ -1,16 +1,65 @@
...
+            tabBarIcon: () => (
+                <IoniconsIcon name="md-settings" size={26} color="white" style={{marginTop:-2}}/>            
+            )
+        }},
+    },
+    {
+    initialRouteName: 'Summary',
+    tabBarPosition: 'bottom',
+    tabBarOptions: {
+        activeTintColor:'white',
```
Comment (seendsouza) [inline src/navigation/AppNavigator.js:38]: Replace all of these with theme.colors.white

File: src/screens/Events.tsx [hunk 8]
```diff
@@ -71,12 +73,16 @@ class Events extends React.Component<Props, {}> {
...
-        <Block flex={false} row style={{paddingVertical: 15}}>
+          <Block flex={false} row style={{paddingVertical: 15}}>
           <Block center>
-            <Text h3 white style={{marginRight: -(25 + 5)}}>
+            <Text h3 white style={{fontSize:21, marginRight: -(100 + 10 + 30)}}>
               Events
             </Text>
           </Block>
+          <View style={{width:100, marginRight:10}}>
+            <Text h4 white style={{fontSize:10, textAlign:'right'}}>Eshwar C.</Text>
```
Comment (seendsouza) [inline src/screens/Events.tsx:84]: Put this data in mocks like the others and use Redux to get it from mocks like the others. 
```javascript
const users = [
    {
        id:1,
        userName: 'eshwarc',
        fullName: "Eshwar Chockalingom"
    }
]
```

Comment (seendsouza) [reply src/screens/Events.tsx:84]: You can create a branch of dev, change the base of this pr to that branch and do this in a later commit if you wish

File: src/screens/Settings.tsx [hunk 9]
```diff
@@ -0,0 +1,110 @@
...
+  }
+
+  type Props = OwnProps;
+
+export default class Settings extends React.Component<Props, {}> {
+    public static defaultProps = {
+        user: mocks.user,
+        chart: mocks.chart,
+      };
+    renderChart() {
```
Comment (seendsouza) [inline src/screens/Settings.tsx]: remove this function and everything that isn't just that text component saying this is settings

File: src/screens/MatchDetails.tsx [hunk 10]
```diff
@@ -0,0 +1,141 @@
...
+              <TouchableOpacity activeOpacity={0.8} onPress={()=>this.props.navigation.navigate('Matches')} style={styles.back}> 
+            <IoniconsIcon name="ios-arrow-round-back" size={45} color="white"/>            
+          </TouchableOpacity>
+              <Block center>
+              <Text h3 white style={{fontSize:19, marginRight: -(100 + 10 + 30 - 50 - 10), marginTop:3,}}>
+                  Match Details
+                </Text>
+              </Block>
+              <View style={{width:100, marginRight:10}}>
+                <Text h4 white style={{fontSize:10, textAlign:'right'}}>Eshwar C.</Text>
```
Comment (seendsouza) [inline src/screens/MatchDetails.tsx:76]: Same as before get from redux

File: src/screens/Summary.tsx [hunk 11]
```diff
@@ -0,0 +1,110 @@
...
+  }
+
+  type Props = OwnProps;
+
+export default class Summary extends React.Component<Props, {}> {
+    public static defaultProps = {
+        user: mocks.user,
+        chart: mocks.chart,
+      };
+    renderChart() {
```
Comment (seendsouza) [inline src/screens/Summary.tsx]: Same as before remove everything but the Text component saying this is Summary page

**Review by seendsouza — CHANGES_REQUESTED** [review]

**Review by seendsouza — APPROVED** [review]

**Pull request closed without merging**